src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -419,6 +473,8 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         // settings might be out-of-proc in the future
         auto settings{ _core.Settings() };
 
+        _useRightClickContextMenu = settings.RightClickContextMenu();
+

```

Comment:
Do we use `_useRightClickContextMenu` anywhere? Why can't we call `_settings.RightClickContextMenu()` when we need it?